modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -762,16 +762,13 @@ public CompletableFuture<Replica> startReplica(
 
     /**
      * Creates and start new replica.
-     * TODO: must be deleted or be private after https://issues.apache.org/jira/browse/IGNITE-22373
      *
      * @param replicaGrpId Replication group id.
      * @param storageIndexTracker Storage index tracker.
      * @param newReplicaListenerFut Future that returns ready ReplicaListener for replica creation.
      * @return Future that promises ready new replica when done.
      */
-    @VisibleForTesting
-    @Deprecated
-    public CompletableFuture<Replica> startReplica(
+    private CompletableFuture<Replica> startReplica(
```

Comment:
It's still @Deprecated @VisibleForTesting and should be removed in https://issues.apache.org/jira/browse/IGNITE-22373. The only change for now is public -> private.

Reply:
Yes, it will be removed an the current state of 22373's patch [reflects it](https://github.com/apache/ignite-3/pull/4144/files#diff-69e9a603ac4198f678f9d8168888bfa70292521abcc8a22373e3114062e4fc2cL734-L820).

Reply:
I mean that formally speaking this method is still @Deprecared and @VisibleForTesting, thus it's incorrect to remove given annotations.